jenkins-blood.planx-pla.net/manifest.json

```diff
@@ -314,6 +315,116 @@
     "config_index": "jenkins_configs_alias",
     "auth_filter_field": "auth_resource_path"
   },
+  "mariner": {
+    "storage": {
+      "s3": {
+        "name": "workflow-engine-garvin",
+        "region": "us-east-1"
+      }
+    },
+    "secrets": {
+      "awsusercreds": {
+        "name": "workflow-bot-g3auto",
+        "key": "awsusercreds.json"
+      }
+    },
+    "jobs": {
+      "engine": {
+        "labels": {
+          "app": "mariner-engine"
+        },
+        "serviceaccount": "mariner-service-account",
+        "restart_policy": "never"
+      },
+      "task": {
+        "labels": {
+          "app": "mariner-task"
+        },
+        "restart_policy": "never"
+      }
+    },
+    "containers": {
+      "engine": {
+        "name": "mariner-engine",
+        "image": "quay.io/cdis/mariner-engine:feat_gwas-pilot-races",
```

Comment:
I guess we're keeping this test image instead of going for master as we don't expect it to change? 🤷

Reply:
I created this draft from qa-mickey for future consumption, copied this block from qa-mickey for now since that works ... when it is ready to merge, some changes would be needed